<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YieldCurveWindow</class>
 <widget class="QWidget" name="YieldCurveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1615</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yield Curve</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QWidget" name="chartWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="chartGridLayout">
      <item row="0" column="0">
       <widget class="QWidget" name="chartPlaceholderWidget" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_AdjustYaxis">
        <property name="text">
         <string>Adjust Y-axis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
